src/style.css

```
@@ -0,0 +1,87 @@
+
+.body {
+  margin: 0;
+}
+
+.snake {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.snake-tile {
+  height: 300px;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+}
+
+.snake-tile:first-child {
+  background-color: #f00;
+}
+
+.snake-tile:nth-child(2) {
+  background-color: #c00;
+}
+
+.snake-tile:nth-child(3) {
+  background-color: #900;
+}
+
+.snake-tile:nth-child(4) {
+  background-color: #600;
+}
+
+.snake-tile:nth-child(5) {
+  background-color: #300;
+}
+
+.snake-tile:nth-child(6) {
+  background-color: #000;
+}
+
+@media (min-width:900px) {
+  .snake-tile:nth-child(n) {
+    flex-basis: 33%;
+  }
+
+  .snake-tile:nth-child(4) {
+    order: 2;
+  }
+
+  .snake-tile:nth-child(5) {
+    order: 1;
+  }
+}
+
+@media (max-width:899px) {
+  .snake-tile:nth-child(n) {
+    flex-basis: 50%;
+  }
+
+  .snake-tile:nth-child(3) {
+    order: 2;
+  }
+
+  .snake-tile:nth-child(4) {
+    order: 1;
+  }
+
+  .snake-tile:nth-child(5) {
+    order: 3;
+  }
+
+  .snake-tile:nth-child(6) {
+    order: 4;
+  }
+}
+
+@media (max-width:599px) {
+  .snake-tile:nth-child(n) {
+    order: 0;
+    flex-basis: 100%;
+  }
```

```suggestion
  .snake-tile {
    order: 0;
    flex-basis: 100%;
  }
```
`:nth-child(n)` has no sense 😉